package.json

```diff
@@ -9,6 +9,7 @@
     "babel-polyfill": "^6.0.16",
     "babel-runtime": "^5.8.25",
     "classnames": "^2.2.0",
+    "escape-string-regexp": "^1.0.5",
```

Comment:
My bad. I copied this dependency from BF (which don't use lodash). I will fix it.